Bottom: 7ff4dd3f9a273862b995721c80c5d7de75b3b1ed
Top:    6c3048c884ad8ad307dc6d52ccc612b7e74b33b1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-22 18:21:25 +0000

Refresh of create-c-remove-element-parame

---

diff --git a/src/copy.c b/src/copy.c
index aaad13e..6c61412 100644
--- a/src/copy.c
+++ b/src/copy.c
@@ -121,7 +121,7 @@ CopyElementLowLevel (DataType *Data, ElementType *Src,
   ElementType *Dest;
 
   /* both coordinates and flags are the same */
-  Dest = CreateNewElement (Data, NULL, &PCB->Font,
+  Dest = CreateNewElement (Data, &PCB->Font,
 			   MaskFlags (Src->Flags, mask_flags),
 			   DESCRIPTION_NAME (Src), NAMEONPCB_NAME (Src),
 			   VALUE_NAME (Src), DESCRIPTION_TEXT (Src).X + dx,
